### PR TITLE
Add documentation for drag/fling offset in WidgetController.

### LINKED ISSUE #56150: WidgetTester documentation doesn't specify if positive y offsets mean up or down
For example, `dragFrom`: https://github.com/flutter/flutter/blob/0549ab239009e6f76f95881364e935cd345a4da7/packages/flutter_test/lib/src/controller.dart#L467

Documentation says:
```
  /// Attempts a drag gesture consisting of a pointer down, a move by
  /// the given offset, and a pointer up.
```

Does a positive y mean you are moving your finger lower on the screen? or moving your finger up?

### PR DESCRIPTION
**Pull request by peixinli**
Add documentation for `offset` in WdigetController to explain the direction on positive/negative coordinate values

Fixes #56150

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: packages/flutter_test/lib/src/controller.dart [hunk 1]
```diff
@@ -562,6 +562,16 @@ abstract class WidgetController {
   ///
   /// {@macro flutter.flutter_test.WidgetController.tap.warnIfMissed}
   ///
+  /// {@template flutter.flutter_test.WidgetController.fling.offset}
+  /// The `offset` represents a distance the pointer moves in global coordinate
```
Comment (Piinks) [inline packages/flutter_test/lib/src/controller.dart:566]:
```suggestion
  /// The `offset` represents a distance the pointer moves in the global coordinate
```

File: packages/flutter_test/lib/src/controller.dart [hunk 2]
```diff
@@ -562,6 +562,16 @@ abstract class WidgetController {
...
+  /// system of the screen.
+  ///
+  /// Positive [Offset.dy] values mean pointer moves downward. Negative
```
Comment (Piinks) [inline packages/flutter_test/lib/src/controller.dart:569]:
```suggestion
  /// Positive [Offset.dy] values mean the pointer moves downward. Negative
```

File: packages/flutter_test/lib/src/controller.dart [hunk 3]
```diff
@@ -562,6 +562,16 @@ abstract class WidgetController {
...
+  /// [Offset.dy] values mean pointer moves upwards. Accordingly, positive
```
Comment (Piinks) [inline packages/flutter_test/lib/src/controller.dart:570]:
```suggestion
  /// [Offset.dy] values mean the pointer moves upwards. Accordingly, positive
```

File: packages/flutter_test/lib/src/controller.dart [hunk 4]
```diff
@@ -562,6 +562,16 @@ abstract class WidgetController {
...
+  /// [Offset.dx] values mean pointer moves towards the right. Negative
```
Comment (Piinks) [inline packages/flutter_test/lib/src/controller.dart:571]:
```suggestion
  /// [Offset.dx] values mean the pointer moves towards the right. Negative
```

File: packages/flutter_test/lib/src/controller.dart [hunk 5]
```diff
@@ -562,6 +562,16 @@ abstract class WidgetController {
...
+  /// [Offset.dx] values mean pointer moves towards left.
```
Comment (Piinks) [inline packages/flutter_test/lib/src/controller.dart:572]:
```suggestion
  /// [Offset.dx] values mean the pointer moves towards left.
```

**Review by Piinks — COMMENTED** [review]
This is very helpful! I always forget which way is which. Just a few nits for grammar. :)

File: packages/flutter_test/lib/src/controller.dart (hunk 4 above)
Comment (peixinli) [reply packages/flutter_test/lib/src/controller.dart]: Done

File: packages/flutter_test/lib/src/controller.dart (hunk 5 above)
Comment (peixinli) [reply packages/flutter_test/lib/src/controller.dart]: Done

File: packages/flutter_test/lib/src/controller.dart (hunk 3 above)
Comment (peixinli) [reply packages/flutter_test/lib/src/controller.dart]: Done

File: packages/flutter_test/lib/src/controller.dart (hunk 2 above)
Comment (peixinli) [reply packages/flutter_test/lib/src/controller.dart]: Done

File: packages/flutter_test/lib/src/controller.dart (hunk 1 above)
Comment (peixinli) [reply packages/flutter_test/lib/src/controller.dart]: Done

**Review by Piinks — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**